pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -158,22 +160,34 @@ func (o *capability) UnregisterFromWorkflow(ctx context.Context, request capabil
 	return nil
 }
 
-func (o *capability) Execute(ctx context.Context, callback chan<- capabilities.CapabilityResponse, request capabilities.CapabilityRequest) error {
+func (o *capability) Execute(ctx context.Context, callback chan<- capabilities.CapabilityResponse, r capabilities.CapabilityRequest) error {
 	// Receives and stores an observation to do consensus on
 	// Receives an aggregation method; at this point the method has been validated
 	// Returns the consensus result over a channel
-	r, err := o.unmarshalRequest(ctx, request, callback)
+	inputs, err := o.ValidateInputs(r.Inputs)
 	if err != nil {
 		return err
 	}
-	o.lggr.Debugw("Execute - adding to store", "workflowID", r.WorkflowID, "workflowExecutionID", r.WorkflowExecutionID, "observations", r.Observations)
 
-	err = o.store.add(ctx, r)
+	// TODO: Remove this debug log and write a test for this
+	fmt.Println("OCR3Capability Execute inputs", r.Inputs, inputs)
```

Comment:
Remove this and add a test. There isnt a proper failing test when the inputs aren't being unmarshalled correctly